script.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-undef */
+const BookData = document.querySelector('#books');
+const button = document.querySelector('#btn');
+const titleInput = document.querySelector('#title');
+const authorInput = document.querySelector('#author');
+
+// Fire event when the initial HTML document has been completely loaded
+document.addEventListener('DOMContentLoaded', () => {
+  let str = '';
+
+  const showData = () => {
+    if (localStorage.getItem('book') === null) {
+      const booksArray = [];
+      localStorage.setItem('book', JSON.stringify(booksArray));
+    } else {
+      const booksArrayStr = localStorage.getItem('book');
+      booksArray = JSON.parse(booksArrayStr);
+    }
+    booksArray.map((data, index) => {
+      str += `
+              <p>${data[0]}</p>
+              <p>${data[1]}</p>
+              <button onclick='remove(${index})'>Remove</button>
+              <hr>
+          `;
+
+      return (data);
+    });
+    BookData.innerHTML = str;
+  };
+
+  button.addEventListener('click', () => {
+    if (titleInput.value === '' && authorInput.value === '') {
+      alert('Please enter a title and author');
+    } else {
+      const bookTitle = titleInput.value;
+      const bookAuthor = authorInput.value;
+      if (localStorage.getItem('book') === null) {
+        const booksArray = [];
+        booksArray.push([bookTitle, bookAuthor]);
+        localStorage.setItem('book', JSON.stringify(booksArray));
```

Comment:
- [ ] Please use objects to create a book instead of an array to follow the project requirements. You should only use an array to store all the books.
 - Hint: `booksArray.push({title: bookTitle, author: bookAuthor});`